tests/contrib/openstack/test_os_contexts.py

```diff
@@ -2929,6 +2933,66 @@ def test_haproxy_context_with_prometheus_exporter(self, local_unit, local_addres
                                                call('cluster'),
                                                call('haproxy-exporter')])
 
+    @patch('charmhelpers.contrib.openstack.context.local_address')
+    @patch('charmhelpers.contrib.openstack.context.local_unit')
+    def test_haproxy_context_with_data_proxyv2(self, local_unit, local_address):
```

Comment:
consider renaming this to imply testing apache context rather than haproxy

Reply:
this is for haproxy context and for apache, there is another one.